Jamulus.pro

```diff
@@ -93,10 +93,29 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+            !exist("C:/Program Files (x86)") {
+                message ("Cross compilation build")
+                programfilesdir = "C:/Program Files (x86)"
+                !exists("$${"programfilesdir"}/JACK2/include/jack/jack.h") {
```

Comment:
`$${"programfilesdir"}` -- this quoting looks surprising to me, but I don't claim to be a `qmake` expert. I would have expected `$${programfilesdir}` similar to `$${libjackname}` below. Is this necessary, if so, why?

This affects this line and some lines further down.

Reply:
86c02e43